Makefile

```diff
@@ -145,7 +140,7 @@ fail_release:
 # -- https://github.com/{TITAN_REPO}/titan/tree/{TITAN_BRANCH}.
 # Default: TITAN_REPO=pingcap, TITAN_BRANCH=master
 titan_release:
-	UPDATE_TITAN=1 make release
+	cargo build --release --no-default-features --features "${ENABLE_FEATURES} update_titan"
```

Comment:
horrible, why including update_titan by default? 

and why do we have a titan_release?

Reply:
`titan_release` is used by Schrodinger test case to do daily bench/test for latest Titan.

Reply:
In normal circumstances, we would not use this to build binary. This PR is to fix the miswrite